.github/workflows/tests.yml

```diff
@@ -152,4 +152,6 @@ jobs:
         run: uv pip install nox --system
 
       - name: Run examples
+        env:
+          HF_TOKEN: ${{ secrets.HF_TOKEN }}
```

Comment:
@0x2b3bfa0 could you please review if it's safe (I have a budget limit on the token here, just in case) and if we need to add it to the list of credentials / tokens

Reply:
Yes, it is safe (external contributors don't have access), and no, having it listed on GitHub Secrets is enough — more or less — to keep track of it.